docs/b2b-edition/specs/storefront/storefront/order.yaml

```
@@ -1535,536 +1491,922 @@ paths:
+          - display_name_customer
+          - display_name_merchant
+          - display_value
+          - display_value_customer
+          - display_value_merchant
+          - value
+          - type
+          - name
+          - display_style
+    productBrandName:
+      type: string
+      description: The name of the brand associated with the product.
+      example: "Rustic Roots"
+    shipping_method:
+      type: string
+      description: The name of the shipping method.
+      example: "2nd Day Air"
+    orderData_ALL:
+      properties:
+        orderId:
+          $ref: '#/components/schemas/orderId'
+        companyName:
+          $ref: '#/components/schemas/companyName'
+        createdAt:
+          $ref: '#/components/schemas/createdAt'
+        updatedAt:
+          $ref: '#/components/schemas/updatedAt'
+        isInvoiceOrder:
+          $ref: '#/components/schemas/isInvoiceOrder'
+        orderStatus:
+          $ref: '#/components/schemas/orderStatus'
+        customOrderStatus:
+          $ref: '#/components/schemas/customOrderStatus'
+        statusCode:
+          $ref: '#/components/schemas/statusCode'
+        totalIncTax:
+          $ref: '#/components/schemas/totalIncTax'
+        currencyCode:
+          $ref: '#/components/schemas/currencyCode'
+        money:
+          $ref: '#/components/schemas/money'
+        firstName:
+          $ref: '#/components/schemas/firstName'
+        lastName:
+          $ref: '#/components/schemas/lastName'
+        poNumber:
+          $ref: '#/components/schemas/poNumber'
+        referenceNumber:
+          $ref: '#/components/schemas/referenceNumber'
+        channelName:
+          $ref: '#/components/schemas/channelName'
+        extraInt1:
+          $ref: '#/components/schemas/extraInt1'
+        extraInt2:
+          $ref: '#/components/schemas/extraInt2'
+        extraInt3:
+          $ref: '#/components/schemas/extraInt3'
+        extraInt4:
+          $ref: '#/components/schemas/extraInt4'
+        extraInt5:
+          $ref: '#/components/schemas/extraInt5'
+        extraStr1:
+          $ref: '#/components/schemas/extraStr1'
+        extraStr2:
+          $ref: '#/components/schemas/extraStr2'
+        extraStr3:
+          $ref: '#/components/schemas/extraStr3'
+        extraStr4:
+          $ref: '#/components/schemas/extraStr4'
+        extraStr5:
+          $ref: '#/components/schemas/extraStr5'
+        extraText:
+          $ref: '#/components/schemas/extraText'
+        extraInfo:
+          $ref: '#/components/schemas/extraInfo'
+        extraFields:
+          $ref: '#/components/schemas/extraFields'
+    responseObject:
+      type: object
+      properties:
+        code:
+          type: integer
+          description: "HTTP Response Code"
+          default: '200'
+        message:
+          type: string
+          description: "Response message"
+          default: 'SUCCESS'
+        data:
+          type: object
+    responseSuccess:
```

```suggestion
    responseObject:
      type: object
      properties:
        code:
          type: integer
          description: "HTTP Response Code"
          default: '200'
        message:
          type: string
          description: "Response message"
          default: 'SUCCESS'
        data:
          type: object
    responseArray:
      type: object
      properties:
        code:
          type: integer
          description: "HTTP Response Code"
          default: '200'
        message:
          type: string
          description: "Response message"
          default: 'SUCCESS'
        data:
          type: array
          items:
            type: object
    responseSuccess:
```
Adding this schema to support the other three suggestions.
Without these four adjustments, it works fine in Swagger, but our own renderer gets lost in the muck.